wagtail/images/models.py

```diff
@@ -185,12 +185,54 @@ def open_file(self):
     @contextmanager
     def get_willow_image(self):
         with self.open_file() as image_file:
-            yield WillowImage.open(image_file)
+            yield willow.Image.open(image_file)
+
+
+class ImageFieldFile(models.fields.files.ImageFieldFile):
```

Comment:
Personally I'd prefer to name these `WagtailImageFieldFile` and `WagtailImageField` for minimum ambiguity

Reply:
None of the other classes in the file are prefixed with Wagtail. It felt inconsistent to prefix the classes themselves rather the depend on the name space to disambiguate.